pkg/lint/lintersdb/manager.go

```diff
@@ -237,18 +237,18 @@ func (m Manager) GetAllEnabledByDefaultLinters() []linter.Config {
 	return ret
 }
 
-func linterConfigsToMap(lcs []linter.Config) map[string]*linter.Config {
+func linterConfigsToMap(lcs []*linter.Config) map[string]*linter.Config {
 	ret := map[string]*linter.Config{}
 	for _, lc := range lcs {
 		lc := lc // local copy
```

Comment:
@jirfag Is this local copy necessary if we use pointers?

Reply:
it's for `scopelint` - it finds a lot of false-positives